features/step_definitions/networking.rb

```diff
@@ -482,12 +497,39 @@
   ensure_admin_tagged
   _admin = admin
 
-  if env.version_ge("3.10", user: user)
-    sdn_pod = BushSlicer::Pod.get_labeled("app=sdn", project: project("openshift-sdn", switch: false), user: admin) { |pod, hash|
+  _host = node.host
+  @result = _host.exec_admin("systemctl is-enabled openvswitch")
+  if @result[:success] && (@result[:response].include? "enabled")
+    logger.info("environment using systemd to launch openvswitch")
+    ovs_restart_timeout = 30
+    wait_for(ovs_restart_timeout) {
+      @result = _host.exec_admin("systemctl status ovsdb-server")
+      if @result[:success] && @result[:response] =~ /Active:\s+?active/
+        logger.info("openvswitch active")
+        true
+      else
+        false
+      end
```

Comment:
@akostadinov  I'm not sure if this is a proper wait loop for `exec_admin` commands.  I'll look for more examples.